control/connman.py

```
@@ -0,0 +1,42 @@
+import logging
+import dbus
+from dbus.exceptions import DBusException
+
+CONNMAN_SERVICE_NAME = 'net.connman'
+CONNMAN_MANAGER_INTERFACE = '%s.Manager' % CONNMAN_SERVICE_NAME
+CONNMAN_SERVICE_INTERFACE = '%s.Service' % CONNMAN_SERVICE_NAME
+
+class ConnmanClient(object):
+    def __init__(self):
+        self.bus = dbus.SystemBus()
+
+        self.manager = self._get_manager_interface()
+        if not self.manager:
+            logging.error('Unable to get connman manager. Probably \
+                Connman is not running')
+            return
```

Logging the error is not enough.
As having a defined `self.manager` is mandatory, this should raise an exception.
Maybe you should also retry before aborting.